MantidQt/MantidWidgets/src/DataProcessorUI/AbstractDataProcessorTreeModel.cpp

```diff
@@ -0,0 +1,38 @@
+#include "MantidQtMantidWidgets/DataProcessorUI/AbstractDataProcessorTreeModel.h"
+#include "MantidAPI/ITableWorkspace.h"
+#include "MantidAPI/TableRow.h"
+
+namespace MantidQt {
+namespace MantidWidgets {
+using namespace Mantid::API;
+
+//----------------------------------------------------------------------------------------------
+/** Constructor
```

Comment:
Details on the object are provided in the header file, which I have now updated a bit.